airflow-core/tests/unit/serialization/test_dag_serialization.py

```diff
@@ -3199,6 +3318,8 @@ def test_handle_v1_serdag():
     v1["dag"]["catchup"] = False
     v1["dag"]["disable_bundle_versioning"] = False
 
+    serialized_simple_dag_ground_truth["dag"]["dag_dependencies"] = expected_dag_dependencies
+
     expected = copy.deepcopy(serialized_simple_dag_ground_truth)
```

Comment:
```suggestion
    expected = copy.deepcopy(serialized_simple_dag_ground_truth)
    expected["dag"]["dag_dependencies"] = expected_dag_dependencies
```

So we don't affect the global variable.